README.md

```diff
@@ -145,34 +151,39 @@ $ aim up
     - [down](#down)
     - [upgrade](#upgrade)
     - [pull](#pull)
+- [Features](#features)
+  - [Searching Experiments](#searching-experiments)
+    - [Search Examples](#search-examples)
   - [TensorBoard Experiments](#tensorboard-experiments)
   - [Anonymized Telemetry](#anonymized-telemetry)
   - [Contributor Guide](https://github.com/aimhubio/aim/wiki/Contributing)
 
-## Installation
-To install Aim, you need to have python3 and pip3 installed in your environment
-1. Install Aim python package
-```shell
-$ pip install aim
-```
-In order to start Aim UI you need to have Docker installed.
-```shell
-$ aim up
-```
+# Introduction
+
+We've written this document to help you navigate through Aim specs and give you a good foundational knowledge about the tool.
+
+## Glossary
+
+We tried to keep Aim as simple as possible. Still, here are a few details to avoid confusion:
 
-## Concepts
-Aim doesn't use too many concepts that needs explaining. Still, here are a few details to avoid confusion:
 - **Experiment**: it's a grouping mechanism for training runs. Experiments are a way to organize the training runs in a meaningful manner
 - **Run**: it's the single training run researchers start
 
 These two are separated just to enable effective organization of the research work.
 
-## Where is the Data Stored 
+## Data Storage 
+
 When the AI training code is instrumented with [Aim Python Library](#python-library) and ran, aim automatically creates a `.aim` directory where the project is located. All the metadata tracked during training via the Python Library is stored in `.aim`.
 Also see [`aim init`](#init) - an optional and alternative way to initialize aim repository.
 
+# Specifications
+
+@gevorg @gor I would like to add also something here, to describe the high level situation, eg tool has 3 main components, they work together, you can use either one of them or all, etc. 
```

Comment:
@SGevorg @gorarakelyan can you draft a generic text for specification, I will later format and bring it in order. 
Essentially it needs to say high level situation about the components, or however you call them :D